tests/cmd/auth/test_login.py

```diff
@@ -479,12 +481,12 @@ def prepare_mocks(
 
         # run cli command
 
-    def run_cmd(self, cli_fs_runner):
+    def run_cmd(self, cli_fs_runner, method="web", expect_check_ff=True):
```

Comment:
What does "ff" mean in `expect_check_ff`?

Reply:
feature_flag, I could juste write it.